src/models/topic.test.ts

```diff
@@ -9,16 +9,6 @@ import {
 import { IconNames } from "../constants";
 import { KafkaTopic, KafkaTopicTreeItem } from "./topic";
 
-describe("Test KafkaTopic methods", () => {
-  it("Local topics should smell local", () => {
-    assert.strictEqual(true, TEST_LOCAL_KAFKA_TOPIC.isLocalTopic());
-  });
-
-  it("CCloud topics should not smell local", () => {
-    assert.strictEqual(false, TEST_CCLOUD_KAFKA_TOPIC.isLocalTopic());
-  });
-});
```

Comment:
Moved to https://github.com/confluentinc/vscode/blob/djs/direct-connection-resources/src/models/resource.test.ts